iceoryx2/src/config.rs

```diff
@@ -284,6 +286,38 @@ pub struct Event {
     pub notifier_dead_event: Option<usize>,
 }
 
+/// Default settings for the request response messaging pattern. These settings are used unless
+/// the user specifies custom QoS or port settings.
+#[non_exhaustive]
+#[derive(Serialize, Deserialize, Debug, Clone, Eq, PartialEq)]
+#[serde(rename_all = "kebab-case")]
+pub struct RequestResonse {
+    /// Defines if the request buffer of the [`Service`] safely overflows.
+    pub enable_safe_overflow_for_requests: bool,
```

Comment:
At some point we need to tackle #17

Reply:
When we have finished #17 the config entry will be the same. But the setting will then only be applied to the subscriber or client/server.